GettingStarted.md

```diff
@@ -1,36 +1,36 @@
-# SailfishOS Chum - Getting Started with OBS
+# SailfishOS:Chum - Getting Started with OBS
 
-This document summarises the proces of building your first package on 
-OBS.  It only skims the surface of OBS and is not a manual.
+This document summarises the process of building your first package on the Sailfish&nbsp;OS OBS.
+It only skims the surface of OBS and is not a manual.
 
 To start building your packages on OBS you will require an account.
-Follow the process in the README and contact 'lbt' for an account to 
-be set up.
+If you are not using the Sailfish&nbsp;OS OBS yet, you shall obtain an account there by contacting the user `lbt` t the IRC channel `#sailfishos`.
 
-When your account is set up, you will have a "Home Project".  This is where
-you can start creating packages.  The URL for your home project will be:
+When your account is set up, you will have a "Home Project".
+This is where you can start creating packages.
+The URL for your home project will be:<br />
 https://build.sailfishos.org/project/show/home:<username>
```

Comment:
```suggestion
https://build.sailfishos.org/project/show/home:username
```

Reply:
That was not me.  See https://github.com/sailfishos-chum/main/blame/main/GettingStarted.md#L12

But you are right, that this is wrong: It must be escaped.
Will do.

Reply:
Ugh, only `<username\>` works properly, `\<username\>` renders as "\\<username\>" at this location (i.e., in contrast to using `\<username\>` solitarily)!